IDE/Espressif/include.am

```diff
@@ -0,0 +1,23 @@
+# vim:ft=automake
+# included from Top Level Makefile.am
+# All paths should be given relative to the root
```

Comment:
Need to add IDE/Espressif/VisualGDB/wolfssl_IDF_v5.2_ESP32.vgdbproj ?

Reply:
yes, that was missing. too. 

root cause of the pesky git actions error was an off-by-one keypress. I inadvertently had a `$` for a comment in the new `include.am` instead of the adjacent key `#`.